test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -347,30 +338,24 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		})
 
 		It("[test_id:22764] verify that by default RT kernel is disabled", func() {
-			conditionUpdating := machineconfigv1.MachineConfigPoolUpdating
-
 			if profile.Spec.RealTimeKernel == nil || *profile.Spec.RealTimeKernel.Enabled == true {
 				Skip("Skipping test - This test expects RT Kernel to be disabled. Found it to be enabled or nil.")
 			}
-
-			By("Applying changes in performance profile")
 			profile.Spec.RealTimeKernel = nil
-			spec, err := json.Marshal(profile.Spec)
-			Expect(err).ToNot(HaveOccurred())
+			// TODO - Need to confirm MC updation after removal of RTK in HCP
+			if hypershift.IsHypershiftCluster() {
+				Skip("Test Case needs behaviour approval for hypershift")
+			}
+			By("Updating the performance profile")
+			profiles.UpdateWithRetry(profile)
 
-			By("Applying changes in performance profile and waiting until mcp will start updating")
-			Expect(testclient.Client.Patch(context.TODO(), profile,
-				client.RawPatch(
-					types.JSONPatchType,
-					[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, spec)),
-				),
-			)).ToNot(HaveOccurred())
+			By(fmt.Sprintf("Applying changes in performance profile and waiting until %s will start updating", poolName))
+			profilesupdate.WaitForTuningUpdating(context.TODO(), profile)
+
+			By(fmt.Sprintf("Waiting when %s finishes updates", poolName))
+			profilesupdate.WaitForTuningUpdated(context.TODO(), profile)
```

Comment:
The above lines are not required,  the test is checking if mcp update stays to false, On regular openshift when *profile.Spec.RealTimeKernel = nil is set we only check if mcp updating is false, we are not waiting for mcp to update.

Reply:
Improved the test for hypershift under latest commit, Thanks